pkg/validations/upgradevalidations/preflightvalidations.go

```diff
@@ -94,5 +94,17 @@ func (u *UpgradeValidations) PreflightValidations(ctx context.Context) []validat
 			}
 		},
 	}
+
+	if u.Opts.Spec.Cluster.IsManaged() {
```

Comment:
what do u think abt moving the if condition in the `ValidateManagementClusterBundlesVersion` ?

Reply:
I don't have a strong opinion either way. I was following the pattern we established over in the create preflight validations flow: https://github.com/aws/eks-anywhere/blob/368c9b899494f99d17be80adce6599c172b598eb/pkg/validations/createvalidations/preflightvalidations.go#L48-L48

If left up to me I'd probably leave as-is, but again not a strong opinion.